src/Nox.Types.EntityFramework/Types/Month/MonthConverter.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Nox.Types.EntityFramework.Types;
 
-public class MonthToByteConverter: ValueConverter<Month, byte>
+public class MonthConverter: ValueConverter<Month, byte>
 {
-    public MonthToByteConverter() : base(month => month.Value, monthValue => Month.From(monthValue)) { }
+    public MonthConverter() : base(month => month.Value, monthValue => Month.From(monthValue)) { }
```

Comment:
MonthConverter should use "FromDatabase"

Reply:
Required changes were applied